setup.py

```diff
@@ -45,10 +45,12 @@
     "pyusb",
     "ruamel.yaml"
 ]
+
 EXTRAS_REQUIRE = {
-    'VISA': ["pyvisa"]
+    'pyvisa==1.9.0'
```

Comment:
We probably want `pyvisa>=1.9.0` or is it `pyvisa>=1.9.x` ...something like that.